packages/website/lib/utils.js

```diff
@@ -12,6 +12,31 @@ export const formatTimestamp = timestamp => {
   return formattedDate;
 };
 
+/**
+ * Returns the amount in TiBs if the amount is a multiple of 1024, otherwise returns the amount in GiBs
+ * @param {number} amountInGiB
+ * @returns {string}
+ */
+export const formatAsStorageAmount = amountInGiB =>
+  amountInGiB % 1024 === 0 ? `${amountInGiB / 1024} TiB` : `${amountInGiB} GiB`;
+
+/**
+ * Formats an amount (in dollars) as a currency
+ * @param {number} amount
+ * @param {boolean} shouldTruncateToDollar
+ * @returns {string}
+ */
+export const formatCurrency = (amount, shouldTruncateToDollar = false) => {
+  if (isNaN(amount)) return '';
+
+  const formatter = new Intl.NumberFormat('en-US', {
```

Comment:
```suggestion
  const formatter = new Intl.NumberFormat(undefined, {
```

Do we want to always use that locale? I thought the point of `Intl` was to use the locale of the user agent (browser)